pxb/jenkins/pxb-package-testing.groovy

```diff
@@ -0,0 +1,185 @@
+library changelog: false, identifier: 'lib@master', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/Percona-Lab/jenkins-pipelines.git'
+]) _
+
+product_action_playbooks = [
+    pxb80: [
+        install: 'pxb_80.yml',
+        upgrade: 'pxb_80_upgrade.yml',
+        upstream: 'pxb_80_upstream.yml',
+        tarball: 'pxb_80_tarball.yml'
+    ],
+    pxb24: [
+        install: 'pxb_24.yml',
+        upgrade: 'pxb_24_upgrade.yml',
+        upstream: 'pxb_24_upstream.yml',
+        tarball: 'pxb_24_tarball.yml'
+    ]
+]
+
+setup_centos_package_tests = { ->
+    sh '''
+        sudo yum install -y epel-release
+        sudo yum -y update
+        sudo yum install -y ansible
+    '''
+}
+
+setup_stretch_package_tests = { ->
+    sh '''
+        sudo apt-get update
+        sudo apt-get install -y dirmngr gnupg2
+        echo "deb http://ppa.launchpad.net/ansible/ansible/ubuntu trusty main" | sudo tee -a /etc/apt/sources.list > /dev/null
+        sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 93C4A3FD7BB9C367
```

Comment:
As I remember correctly we had the same dialog about trusty and debian 9 on previous review. Why package from stretch repo doesn't work for you?

Reply:
Well, for the same reason it didn't work on the previous review: https://github.com/Percona-Lab/jenkins-pipelines/pull/986#discussion_r642444604

> I could change it so that Buster uses 2.7.7 from the Debian repos instead _[note: and I did]_, but I can't make the same change for Stretch as our playbooks don't run on 2.2.1.

The same applies to the playbooks that are run by this job:

```
> apt-get install -y ansible
> ansible-playbook --version
ansible-playbook 2.2.1.0
> git clone --depth 1 https://github.com/percona-qa/package-testing
> ansible-playbook --connection=local --inventory 127.0.0.1, --limit 127.0.0.1 package-testing/playbooks/pxb_80.yml
ERROR! no action detected in task. This often indicates a misspelled module name, or incorrect module path.
```